.eslintignore

```diff
@@ -6,4 +6,6 @@ node_modules
 /lib
 /esm
 /cjs
-/browser
\ No newline at end of file
+/browser
+esm
+cjs
```

Comment:
Aren't those already present in the file? 🤔

Reply:
Isn't this file already in https://github.com/asyncapi/parser-js/pull/1008/files#diff-bffba01392956f96fbbb518b6549cffcbd4d10587204e4c302c1335c4f644fc9 ?

Reply:
ok removed but added the correct path. I was thinking to use seprate files for each package.